markdown/1.x/en/documentation/currency-guide.md

```diff
@@ -91,24 +87,22 @@ If you are importing multiple price fields like discount prices or something sim
 `<ff-communication currency-fields="discountPrice">`
 
 ---
-#### **currency-min-digits (String)**
-Specify the minimum number of digits. Defaults to `2`.
+#### **currency-min-digits (String) (default: 2)**
+Specify the minimum number of digits.
 
 **Example Usage**
 `<ff-communication currency-min-digits="1">`
 
 ---
-#### **currency-max-digits (String)**
-Specify the maximum number of digits for all prices. Defaults to `2`.
+#### **currency-max-digits (String) (default: 2)**
+Specify the maximum number of digits for all prices.
 
 **Example Usage**
 `<ff-communication currency-max-digits="4">`
 
 ---
-#### **add-unit-to-filter-groups (Boolean)**
-If present, the unit formatting takes place for all filter groups and it's filters. Note that this is a `Boolean` attribute and is therefore ignoring its value. Read more about [attributes](todo) 
-
-**Example Usage**
-`<ff-communication add-unit-to-filter-groups>`
+#### **add-unit-to-filter-groups (Boolean) (deprecated since 1.3.1)**
+~~If present, the unit formatting takes place for all filter groups and its filters.~~
+Since 1.2.2 the formatting takes places for all filter groups and its filters internally for every response. Setting the attribute was a noop or could lead to false encoding in case of the currency value differs on client and FACT-Finder site. Since 1.3.1 this attribute is always a noop.
```

Comment:
Got some grammatical issues.
```
Since 1.2.2, for every response, the formatting is applied internally to all filter groups and their filters. [...] Since 1.3.1 this attribute has no effect.
```
Not sure about how to best express the middle part. Maybe something like:
```
[...] There was a bug that caused the attribute to either have no effect at all or cause wrong encoding under certain circumstances. [...]
```